app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -45,9 +53,28 @@ def check_import_supported!(source_provider)
     end
   end
 
+  def queue_import_vm(userid, source_vm_id, target_params)
+    task_options = {
+      :action => 'Import VM',
+      :userid => userid
+    }
+
+    queue_options = {
+      :zone        => self.my_zone,
+      :class_name  => self.class.name,
+      :method_name => 'import_vm',
+      :instance_id => self.id,
+      :role        => 'ems_operations',
+      :args        => [source_vm_id, target_params]
```

Comment:
@matobet 
Can you add :task_id = nil here for the work to be picked up right away. If you dont have a nil it uses the previous task_id and will block